garak/probes/misleading.py

```diff
@@ -6,7 +6,6 @@
 deny, refute, or contradict.
 """
 
-
```

Comment:
I wonder if we have `black` config variations that disagree about the number of blank lines placed where